src/lib.rs

```diff
@@ -34,6 +34,8 @@
 #![feature(alloc_error_handler)]
 #![feature(vec_into_raw_parts)]
 #![feature(drain_filter)]
+#![feature(llvm_asm)]
```

Comment:
I just skimmed over the commit log so I haven't had a detailed look yet, but what is the context here? Is this just because some llvm_asm code already exists, or did you plan to add new llvm_asm code?

Reply:
After Skimming over the changes, yes this is because there is a lot of llvm_asm in the `aarch64` code. And I think that is perfectly fine for now.

Reply:
Yes, it is currently a workaround for llvm_asm support